instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/SpanNames.java

```diff
@@ -5,10 +5,21 @@
 
 package io.opentelemetry.instrumentation.api.tracer;
 
+import io.opentelemetry.instrumentation.api.caching.Cache;
 import java.lang.reflect.Method;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public final class SpanNames {
+
+  private static final ClassValue<Cache<String, String>> spanNameCaches =
```

Comment:
Can you check whether instrumentation have their own cache to not double cache?

Reply:
sure, just looked through all callers, didn't find any secondary caches